spec/data/sig_handler.txt

```diff
@@ -211,6 +220,12 @@ class CollectionSigs
   sig {returns([String, Integer])}
   def fixed_array; ['', 0]; end
 
+  sig {params(arr: [Foo, Bar]).returns([])}
+  def fixed_empty_array(arr); []; end
+
+  sig {returns(T.any(T::Array[String], []))}
```

Comment:
What do you think of updating this example to not use a redundant union? (`[]` is already a valid `T::Array[String]`)

Reply:
lol I took this straight from the code erroring on yard doc generation. You're right, sorbet eagerly condenses types so this almost immediately is just `T::Array[String]`, contravariance of the returned side. I guess I'm still seeing a point in keeping it as this was the in-the-wild problem sig and it's not like it gets condensed before yard-sorbet sees it. It at least covers the flow of going through `convert_t_method` on `:any` to `convert_array`

But I'm not wedded to it so happy to remove it if you'd prefer it being gone.

Reply:
I don't want to remove it, just replace it with a non-redundant `sig` that might plausibly make it into production, how about `sig {returns(T.any([String], []))}`?